roles/ovirt-host-deploy-firewalld/vars/firewalld_rules_common.yml

```diff
@@ -0,0 +1,9 @@
+host_deploy_firewalld_rules:
+  ## Service definition:
+  - service: rpc-bind
+  - service: snmp
+  - service: cockpit
+
+  ## Port definition:
+  # ovirt-imageio-daemon
+  - port: 54322/tcp
```

Comment:
Is this the version available in CentOS 7.4 and Fedora 25/26? If not we still need patch for ovirt-imageio-daemon